### PR TITLE
Added Android Design Support Library for #258

### PR DESCRIPTION
**Pull request by SecUpwN**
This PR adds the dependency for the new `Android Design Support Library` as described [here](http://android-developers.blogspot.de/2015/05/android-design-support-library.html).

### CODE REVIEW

File: AIMSICD/build.gradle [hunk 1]
```diff
@@ -87,6 +87,7 @@ dependencies {
     
     // Android Support Libraries
     compile 'com.android.support:appcompat-v7:23.1.1'
+    compile 'com.android.support:design:22.2.0'
```
Comment (larsgrefer) [inline AIMSICD/build.gradle]: The version should match with the other support-libraries and our compileSdk

**Comment by larsgrefer** [comment]
@SecUpwN Why do you add the dependency without using it?

**Comment by SecUpwN** [comment]
>Why do you add the dependency without using it?

Because I felt like it would be a small yet inevitable step in moving towards solving #258. You disagree?

**Comment by larsgrefer** [comment]
From the technical point of view this would be completely senseless. From the psychological point of view I agree with you

**Comment by SecUpwN** [comment]
>From the technical point of view this would be completely senseless.

I can close this PR if you think it really makes no sense. Do you?

>From the psychological point of view I agree with you

Great to know soeone feels with me. I feel a little trapped now..

**Comment by larsgrefer** [comment]
IMHO the dependency should be added when its actually used and needed

**Pull request closed without merging**